### PR TITLE
Pull request

### PR DESCRIPTION
**Pull request by OskarHallberg**
Please papa Harry can u review?

### CODE REVIEW

File: Find.java [hunk 1]
```diff
@@ -5,6 +5,10 @@
 public class Find {
   public static void main(String[] args) {
 
+    for(int i = 0; i < 10000000;i++){
```
Comment (itggot-Harry-Norrman) [inline Find.java]: This is an unnecessary loop which clogs up the program, it can be removed

**Review by itggot-Harry-Norrman — CHANGES_REQUESTED** [review]
unnecessary commit

File: Find.java (hunk 1 above)
Comment (OskarHallberg) [reply Find.java]: So soryy papa harry I will do better next time

**Review by itggot-Harry-Norrman — APPROVED** [review]
good job very efficient

**Pull request merged**